cypress/integration/1-getting-started/Local tests/Local_1_Create_a_user_in_a_Corteza_instance.spec.js

```diff
@@ -6,17 +6,14 @@ it('should be able to visit the page', () => {
 })
 
 it('should be able to go to the create a new account page', () => {
-    cy.request(baseURL + '/auth/signup')
-    cy.visit(baseURL + '/auth/signup')
+  cy.request("http://localhost:3000" + '/auth/signup')
```

Comment:
baseURL please, also make sure all other URLs that were changed to work for local use the related (/compose/) prefix in the committed tests